playwright.config.ts

```diff
@@ -0,0 +1,27 @@
+import { PlaywrightTestConfig, devices } from '@playwright/test'
+
+const config: PlaywrightTestConfig = {
+  forbidOnly: !!process.env.CI,
+  retries: process.env.CI ? 2 : 0,
+  reporter: process.env.CI ? [['github'], ['json', { outputFile: 'playwright-report.json' }]] : 'list',
+  testDir: './test',
+  use: {
+    trace: 'on-first-retry',
+    baseURL: 'http://localhost:3000',
+  },
+  timeout: 5000,
+  webServer: {
+    command: 'npm run dev',
+    url: 'http://localhost:3000',
+    cwd: './test',
+    reuseExistingServer: !process.env.CI,
+  },
+  projects: [
```

Comment:
Why not testing for other browsers ?

Reply:
tbh I just pushed some config from another project. I was just trying to bring some initial config to start writing some tests.

which browsers should we be adding?